ccip-cli/src/commands/utils.ts

```diff
@@ -295,6 +296,7 @@ export async function prettyRequest(this: Ctx, source: Chain, request: CCIPReque
       : formatData('data', request.message.data)),
     ...('accounts' in request.message ? formatArray('accounts', request.message.accounts) : {}),
     ...rest,
+    'CCIP Explorer': getCCIPExplorerUrl('msg', request.message.messageId),
```

Comment:
Not sure if this would look good in the table, since it tends to break long lines in different items;
Maybe it'd be better as its own line before or after the table?